src/core/models/payloads/tagged_data.h

```diff
@@ -17,6 +17,7 @@
  * The payload type of tagged data is 5
  */
 typedef struct {
+  uint32_t type;     ///< Set to value 5 to denote a Tagged Data payload.
```

Comment:
the `type` is a constant 5, why do we need it in this struct?
```suggestion

```

Reply:
`Type` has been removed from a structure. Thanks.